### DISCUSSION TITLE
Change audio output device with `setSinkId`

### DISCUSSION THREAD

**Discussion by adriaanwm** [vimeo/player.js#671] [Ideas]
### Expected Behavior

In the embed options it would be nice to have a sinkId option to choose which speakers should be used for audio output. Also a `setSinkId` on the player would be useful to change this value for an active player.

### Actual Behavior

There is no sinkId or setSinkId option, so no way to change the speakers being used.

### Steps to Reproduce

Missing feature. 

`setSinkId` documentation https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/setSinkId

**Comment by luwes** [comment]
thanks for the feedback @adriaanwm. this is quite an exotic API method and I don't think it will be added very soon unless there are many people requesting it. I'll leave it open here to gain popularity but for now if you really need this functionality a 3rd party player is probably the best route.

https://vimeo.zendesk.com/hc/en-us/articles/224823567-Direct-links-to-video-files

**Comment by teemzero** [comment]
Would be nice indeed if this can be added!
Thank you

For now I did it like so:

Wait until Plyr is fully initialized and the source is set, then select the video element (in this case I used getElementsByTagName because the id was used by Plyr and then removed apparently) before firing play();

```
const my_video = document.getElementsByTagName("video")[0];
my_video.setSinkId(YOUR_SINK_ID);
```
